spec/models/service_template_spec.rb

```diff
@@ -240,13 +251,19 @@
     end
 
     it "with service_type of unknown" do
```

Comment:
@bzwei The `service_type` "unknown" is no longer a concern here.  Should this test be removed?

Reply:
The code still accept any string to `service_type`. So the test here is ok although not important.